CONTRIBUTION.md

```diff
@@ -0,0 +1,148 @@
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/kubesimplify/website/pulls)
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/kubesimplify/)
+
+At Kubesimplify, we warmly welcome contributions through collaboration. We are excited to see that you want to contribute!. There are many ways in which one could contribute to Kubesimplify and every contribution is equally appreciated here. Navigate through the following to understand more about contributing here.
+
+# Before you get started
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting [our discord community](https://kubesimplify.com/discord)
+Violation of the code of conduct is taken seriously. The comfort and safety of Kubesimplify community members are our priority. Please do well to adhere to the Code of Conduct to participate in the Kubesimplify community.
+
+# New to Git
+
+Follow these resources: https://lab.github.com and https://try.github.com/
+
+# Style guide
+
+The Kubesimplify website is hosted in this repository and is built using [Docusaurus2.0](). Once a pull request has been submitted, a preview deployment will be built and made available to you and other contributors on your PR to review.
```

Comment:
Should we give link to Docusaurus2.0 here as well?

Reply:
I think Docusaurus shouldn't be mentioned as there is an config.json file, IMHO, which is a key component of docusaurus framework. It mention of Docusaurus would help developers, know about it and work accordingly.

About DCO: I think it's becoming a SOP in open source projects to include DCO. This would help new contributors know about it and get used to. (Maintainers can be linient towards non-signed commits) Which would help them in future OSS contributions.

Above are my views. But, final call is your's.

Reply:
Let the maintainers take the final decision, I am okay with both